minecraft_servers/cytooxien/manifest.json

```diff
@@ -8,7 +8,15 @@
   ],
   "social": {
     "web": "https://www.cytooxien.de/",
+    "web_shop": "https://shop.cytooxien.de/",
+    "web_support": "https://www.cytooxien.de/support/",
     "twitter": "Cytooxien",
-    "discord": "https://cxn.link/discord"
-  }
+    "discord": "https://cxn.link/discord",
+    "teamspeak": "ts.cytooxien.net"
+  },
+  "discord": {
+    "server_id": 443768116216987648,
+    "rename_to_minecraft_name": true
```

Comment:
Please remove the line here too!

Reply:
I guess Cytooxien ist a partner Server 🤔 

https://labymod.net/en/partner

https://i.lennartloesche.de/ROJo3/duBoWaLa91.png

Reply:
Yes. You are very welcome to fill out the `server_id` in (it's good if we have the data). But please take out "`rename_to_minecraft_name`". This is for team members of the server to decide :)